### PR TITLE
Add canary add canary switch

### LINKED ISSUE #11: Canary timeout fails CD/CI, create switch for canary
## Ref
https://github.com/fredrikhgrelland/terraform-nomad-redash/pull/11/checks?check_run_id=1042159073#step:8:401 

## Potential reason
Timeout for canary deployment

```
  update {
    max_parallel = 1
    health_check = "checks"
    min_healthy_time = "10s"
    healthy_deadline = "10m"
    progress_deadline = "12m"
    auto_revert = true
    auto_promote = true
    canary = 1
    stagger = "30s"
  }
```
ref: https://github.com/fredrikhgrelland/terraform-nomad-hive/blob/master/conf/nomad/hive.hcl#L7-L17

### PR DESCRIPTION
**Pull request by zhenik**
## Closes/fixes/resolves issue(s)?
- Closes #11 
## What was added/changed/fixed?
- [added] canary switch
- [update] Makefile
## Checklist (after created PR)
- [x] Added reviewers
- [x] Added assignee(s)
- [x] Added label(s)
- [x] Added to project
- [x] Added to milestone

### CODE REVIEW

**Review by claesgill — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -38,6 +38,7 @@ make test
 | nomad\_datacenters | Nomad data centers | list(string) | ["dc1"] | no |
 | nomad\_namespace | [Enterprise] Nomad namespace | string | "default" | no |
 | local_docker_image | Switch for nomad job | bool | - | yes |
+| use\_canary | Uses canary deployment for Hive | bool | false | no |
 | hive_service_name | Hive service name | string | "hive-metastore" | no |
```
Comment (pdmthorsrud) [inline README.md:42]: Why are some escaped and some are not?

**Review by pdmthorsrud — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (claesgill) [reply README.md]: Why do we even have escaped underscores?

**Review by dangernil — APPROVED** [review]
LGTM

**Review by claesgill — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (pdmthorsrud) [reply README.md]: Not to bring up old wounds, but did we get an answer to this? 😅

**Pull request merged**